tests/installation/src/__tests__/basic/install-api-gen.ts

```diff
@@ -9,9 +9,9 @@
  */
 
 import {
-    checkMandatoryEnvironmentVariables,
-    installAndGenerateApiDocs,
-  } from '../../utils';
+  checkMandatoryEnvironmentVariables, identifySecuritySystem,
```

Comment:
Can you separate `identifySecuritySystem` onto a new line to match the linting of the rest of the variables please